Makefile

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Same here - extra newlines aren't required at the start or end of a file.

Reply:
Sir I will omit them